packages/gestalt/src/NumberField.js

```diff
@@ -24,6 +24,9 @@ type Props = {|
    * Indicate if the input is disabled.
    */
   disabled?: boolean,
+  /**
+   * It specifies what action label to present for the enter key on virtual keyboards.
```

Comment:
The "it" is unnecessary — just "Optionally specify the action label to present…"\

Also: link to the variant?

Reply:
DONE